### PR TITLE
Sean20405 proposal for fftconvolve2d

### PR DESCRIPTION
**Pull request by Sean20405**
---
A good PR should include the following parts:

- [x] Provide a clear title
- [x] Provide meaningful descriptions in the PR and the commit logs
- [x] Name your branch like ``<github-user-name>-hw#``, e.g., ``myname-hw2-attempt1``
- [x] Merge from your specific branch (***not master***) to the `hw*` branch (***not master***, again)
- [x] Exclude irrelevant files, such as binaries, text files, or dot files
- [x] Make sure tests/CI pass

### CODE REVIEW

File: project/Sean20405/README.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+- Students studing image processing
+
+System Architecture
+===================
+
+I/O
+---
+
+-  Input (needed): An array-like object to be convolved and an
+   array-like kernel. The dimension of both inputs should be 2
```
Comment (yungyuc) [inline project/Sean20405/README.rst]: Do you have a definition of "array-like"?

File: project/Sean20405/README.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+-  ``pytest`` is for testing correctness and performance. (compared to
+   the result of the original ``convolve2d()``)
+-  A README will be included for document.
+
+Schedule
+========
+
+Planning phase (11 weeks from 09/29 to 12/08): 
+
+- Week 1 (09/29): Learn FFT
```
Comment (yungyuc) [inline project/Sean20405/README.rst:112]: A fast way to learn is to find a reference code.

File: project/Sean20405/README.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+- Week 2 (10/06): Learn FFT, decide which algorithms would be used
+- Week 3 (10/13): Write test for FFT and FFT convolution
```
Comment (yungyuc) [inline project/Sean20405/README.rst]: Please include setting up Github Actions along with the test development.

**Review by yungyuc — CHANGES_REQUESTED** [review]
To request for a next review, please leave a global message summarizing what you change.

- [x] [L50](https://github.com/yungyuc/nsdhw_24au/pull/43/files#diff-4e4bfc35470ae48b373f517d977aa1ba1be173b65d98d844a742d1c8847f2adcR50): Clarify if you will use a definition for "array-like".
- [x] [L114](https://github.com/yungyuc/nsdhw_24au/pull/43/files#diff-4e4bfc35470ae48b373f517d977aa1ba1be173b65d98d844a742d1c8847f2adcR114): Add Github Actions into the testing plan.

**Comment by Sean20405** [comment]
I'm ready for the second attempt of my proposal. Thanks for such useful feedback!

**Review by yungyuc — CHANGES_REQUESTED** [review]
LGTM.  You need to add an entry in `project/README.rst` pointing to your proposal.

**Comment by Sean20405** [comment]
I'm ready for the third attempt of my proposal.
Thanks for reminder! And I've also resolved the conflict derived from index in `project/README`.

**Review by yungyuc — APPROVED** [review]
Let me know when you want this PR to be merged.

**Pull request merged**